mantle/platform/api/aws/api.go

```diff
@@ -130,3 +132,11 @@ func tagSpecCreatedByMantle(name, resourceType string) []*ec2.TagSpecification {
 		},
 	}
 }
+
+func randomName(prefix string) string {
+	b := make([]byte, 5)
+	if _, err := rand.Read(b); err != nil {
+		plog.Errorf("randomName: failed to generate a random name: %v", err)
+	}
+	return fmt.Sprintf("%s-%x", prefix, b)
+}
```

Comment:
I was thinking more that rather than copy/paste the function from `azure/api.go` we'd put it in `util/` (like the retry functions) and use that everywhere (i.e. we'd delete the copy of this in `azure/api.go` too.

Reply:
Ah, yeah that makes sense. I can make that change as a separate commit. Should I make the change to the azure API in a separate PR after this merges?

Reply:
I would just put it in the same PR.